Userland/Libraries/LibELF/DynamicLoader.cpp

```diff
@@ -240,6 +240,25 @@ void DynamicLoader::do_main_relocations()
                 m_direct_ifunc_relocations.append(relocation);
                 return;
             }
+            if (relocation.type() == R_X86_64_TLSDESC || relocation.type() == R_AARCH64_TLSDESC) {
```

Comment:
Is the `must_bind_now()` branch the correct place for this? Eager binding `-z now` is only enforced in our CMake module, so some ports/manually built programs might still use lazy binding.

(hmm, it would be great to have tests to avoid breaking those).